these come from the toolchain and are not portable, there is no change from adding them..
--
diff --git a/meson.build b/meson.build
index ea26cf1..023ee69 100755
--- a/meson.build
+++ b/meson.build
@@ -24,7 +24,7 @@ lib_c_args = ['-fvisibility=hidden']
 
 # Add default x86 and x86_64 optimizations
 if current_arch == 'x86' or current_arch == 'x86_64' and current_os != 'darwin'
-    lib_c_args += ['-msse','-msse2','-mfpmath=sse','-ffast-math','-fomit-frame-pointer','-fno-finite-math-only']
+    lib_c_args += ['-ffast-math','-fomit-frame-pointer','-fno-finite-math-only']
 endif
 
 
